core/src/main/groovy/noe/common/utils/Platform.groovy

```diff
@@ -152,6 +152,15 @@ class Platform {
     return isHP() && (osName ==~ /.*11.*/)
   }
 
+  boolean isFips() {
+    if(isRHEL()) {
+      // Using return (['/usr/bin/fips-mode-setup', '--is-enabled'].execute().waitFor() == 0)
+      // does not work on RHEL7
+      return (new File('/proc/sys/crypto/fips_enabled').readLines()[0]=='1')
```

Comment:
what will happen if the file is not present or the the array has length 0?

Reply:
if "isRHEL()" the file should be there for the foreseeable past and future, unless /proc is not mounted. But I agree that it should be checked and return false and will modify it.

Reply:
I modified the function by capturing Exception, logging error and returning false